data/tekton-pipelines/okd/windows10-installer.yaml

```diff
@@ -33,15 +34,6 @@ kind: ClusterRole
 metadata:
   name: windows10-pipelines
 rules:
-  - verbs:
```

Comment:
Why is it removed?

Reply:
There are rules defined for datavolumes on line 84, so this seemed like not needed anymore.